### PR TITLE
Hypixel API • Reborn docs is moved to new domain.

### PR DESCRIPTION
**Pull request by StavZ**

### CODE REVIEW

File: websites/H/Hypixel API Reborn/dist/metadata.json [hunk 1]
```diff
@@ -6,12 +6,12 @@
   },
   "service": "Hypixel API Reborn",
   "description": {
-    "en": "Hypixel API • Reborn is the only Hypixel API wrapper for Node.js/Typescript with intellisense support.",
-    "ru": "Hypixel API • Reborn - единственный парсер Hypixel API для Node.js/Typescript с поддержкой intellisense.",
+    "en": "Hypixel API • Reborn is feature-rich Hypixel API wrapper for Node.js.",
```
Comment (Bas950) [inline websites/H/Hypixel API Reborn/dist/metadata.json:9]:
```suggestion
    "en": "Hypixel API • Reborn is a feature-rich Hypixel API wrapper for Node.js.",
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
upload proof of it working

**Comment by StavZ** [comment]
image

**Comment by SlowLife1661** [comment]
Nope we need you to show that Presence is still working on these new domains

**Comment by StavZ** [comment]
image

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**